lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -257,7 +257,7 @@ def define
 
       outcome :outcome_marriage_france_or_french_overseas_territory
 
-      outcome :outcome_os_affirmation
+      outcome :outcome_opposite_sex_marriage_affirmation
```

Comment:
Minor: `outcome_opposite_sex_marriage_in_affirmation_countries` might be better.